### PR TITLE
Show branch change option only to First Year(2nd Semester) B. Tech students.

### PR DESCRIPTION
**Pull request by sanugautam08**
Issue no. #922 
As you can see in the below attached images now branch change option is only available to First Year(2nd Semester) B. Tech student .
Screenshot (684)
Screenshot (685)

### CODE REVIEW

**Comment by sanugautam08** [comment]
Issue no. #926 solved
Removal of repetition of user details from user card.
Screenshot (687)

File: FusionIIIT/templates/globals/usercard.html [hunk 1]
```diff
@@ -33,13 +33,13 @@
                     Sem - {{user.extrainfo.student.curr_semester_no}}
                     {% endif %}
                 </span>
-                <span class="right floated meta">
+                <!--<span class="right floated meta">
```
Comment (garvitgupta13) [inline FusionIIIT/templates/globals/usercard.html]: Instead of commenting please delete this code

**Review by garvitgupta13 — CHANGES_REQUESTED** [review]
Please make the requested changes, then we can merge this pr

File: FusionIIIT/templates/globals/usercard.html (hunk 1 above)
Comment (sanugautam08) [reply FusionIIIT/templates/globals/usercard.html]: Requested changes are done sir.

**Review by garvitgupta13 — APPROVED** [review]
@akanimesh please merge it

**Pull request merged**